src/appleseed/renderer/modeling/aov/diffuseaov.cpp

```diff
@@ -73,18 +75,18 @@ namespace
             const ShadingComponents&    shading_components,
             const float                 multiplier) override
         {
-            m_color = shading_components.m_diffuse;
+            m_color = shading_components.m_diffuse.to_rgb(g_std_lighting_conditions);
             m_color *= multiplier;
         }
 
         virtual void flush(ShadingResult& result) override
         {
-            result.m_aovs[m_index].m_color = m_color;
-            result.m_aovs[m_index].m_alpha = result.m_main.m_alpha;
+            result.m_aovs[m_index].rgb() = m_color;
```

Comment:
`rgb()` returns the first three components (R, G, B) as a `foundation::Color3f` reference so assignment works just fine.